sass/responsive/_mobile.scss

```diff
@@ -1,6 +1,88 @@
 @charset 'UTF-8';
 
 @media screen and (max-width: 768px) {
+    .MenuWrapper {
+        .Menu {
+            &.dropdown-menu {
+                max-height: 100%;
+                max-width: 100%;
+            }
+            &.mobile-channel-header-dropdown-enter {
+                transform: translateY(100%);
+            }
+
+            &.mobile-channel-header-dropdown-enter-active {
+                transform: translateY(0%);
+                transition: transform .35s ease-in;
+            }
+            &.mobile-channel-header-dropdown-enter-done {
+                transform: translateY(0%);
+            }
+
+            &.mobile-channel-header-dropdown-exit {
+                transform: translateY(100%);
+                transition: transform .35s ease-in;
+            }
+        }
+    }
+
+    .post__header {
+        .dropdown-menu {
+            >li {
+                >button {
+                    @include border-radius(0);
+                    border-top: 1px solid;
+                    padding: 15px 20px;
+                }
+
+                &:first-child {
+                    button {
+                        border: none;
+                    }
+                }
+            }
+        }
+    }
+
+    .app__body {
+        .post {
+            .Menu {
+                @include border-radius(0);
+                border: 0px;
+                height: 100%;
+                left: 0;
+                position: fixed;
```

Comment:
Seems like the new scrolling overhaul work is interfering with the mobile menu implementation. `position: fixed` here is no longer relative to the document, but rather the posts container. This is making the mobile menu render above the top of the window for channels with longer conversations. Try out a channel with a couple pages of posts or more.

Thoughts/

Reply:
Seeing the same behaviour on:

- Chrome, Firefox for Mac
- Chrome for Windows
- iPad
- iPhone

Reply:
https://youtu.be/1c5KvOkYWTU

Reply:
@deanwhillier That's an issue in general right, not related to this PR?

Reply:
That's correct. I'm creating a bug ticket now.

Reply:
MM-14820(https://mattermost.atlassian.net/browse/MM-14820)